webapp/backend/urls.py

```diff
@@ -23,6 +23,7 @@
 
 urlpatterns = [
     url(r'^authenticate/?$', views.authenticate),
+    url(r'^user_public_keys/?$', views.user_public_keys),
```

Comment:
Please change this to `user-public-keys` instead of `user_public_keys`. We should keep a consistency in the naming of our urls.